share/picongpu/tests/XrayScattering/etc/picongpu/3.cfg

```diff
@@ -0,0 +1,83 @@
+# Copyright 2013-2020 Axel Huebl, Rene Widera, Felix Schmitt,
+#              Pawel Ordyna
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+##
+## This configuration file is used by PIConGPU's TBG tool to create a
+## batch script for PIConGPU runs. For a detailed description of PIConGPU
+## configuration files including all available variables, see
+##
+##                      docs/TBG_macros.cfg
+##
+
+
+#################################
+## Section: Required Variables ##
+#################################
+
+TBG_wallTime="0:30:00"
+
+TBG_devices_x=1
+TBG_devices_y=3
```

Comment:
3 GPUs is a strange distribution, but okay.

Reply:
@PrometheusPi I think the idea was to have an number of GPUs in the distributed mode that does not divide the number of frequencies in the output. That way I have one GPU with a smaller output.  But since I moved from 512 x 512 to 120x120 output this is useless since 3 divides 120.  I will change it later to 122x122.